src/test/skript/tests/misc/experience to number.sk

```diff
@@ -0,0 +1,3 @@
+test "experience to number converter":
+  set {_xp} to 5 xp
+  assert {_xp} + 5 is 10 with "experience to number converter doesn't work"
```

Comment:
Not overly important but if a experience + number operator is ever added this will break the test